vite.config.js

```diff
@@ -0,0 +1,79 @@
+import { defineConfig } from "vite";
+import { VitePWA } from "vite-plugin-pwa";
+import fs from "fs";
+
+// Transform HTML during build
+function htmlUrlPlugin() {
+  return {
+    name: "html-transform",
+    transformIndexHtml(html) {
+      return html.replace(
+        /(src|href)="\.?\/(images|sound)\//g,
+        `$1="/MinDArT/$2/`
+      );
+    },
+  };
+}
+
+export default defineConfig({
+  base: "/MinDArT/",
```

Comment:
these two lines will need updating if you want to serve the app at a url other than `${your-domain}/MinDArt`: change `MinDArT` to another string (`mindart` is much easier to type, as a url) or change it to `/` if the app will be served at the root of the domain. You may need to double-check the output of the `htmlUrlPlugin` in that case (i.e. check the index.html file in `dist`), and ensure that all the `href` and `src` attributes are being correctly updated to match your intended path.